mantidproject/mantid
# Indirect ILL: Background subtraction from vanadium run

[files.zip](https://github.com/mantidproject/mantid/files/1014766/files.zip)

**To test:**
1. Set ILL/IN16B as instrument
2. Go to Interfaces>Indirect>Data Reduction>Energy Transfer Tab
3. Choose from the uploaded files as follows and click Run:
  - Sample : 090661
  - Calibration : 090663
  - Calibration background : 090658
  - Scaling Factor : 0.1
  - Leave the rest to defaults
4. Output should contain a single QENS peak in each spectra



Fixes #19448 .



---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
